HealthcareSystem/Backend/Model/PerformingExamination/Examination.cs

```diff
@@ -34,6 +35,10 @@ public class Examination
         public virtual PatientCard PatientCard { get; set; }
         public virtual ICollection<Therapy> Therapies { get; set; }
 
+        public virtual ICollection<SurveyAboutDoctor> SurveysAboutDoctor { get; set; }
+        public virtual ICollection<SurveyAboutMedicalStaff> SurveysAboutMedicalStaff { get; set; }
+        public virtual ICollection<SurveyAboutHospital> SurveysAboutHospital { get; set; }
```

Comment:
Doesn't the patient leave just one survey per examination?

Reply:
By accident...
I'll fix it.

Reply:
Fixed in 7ae66b5349986db657ee712ed782bf8d87fc9033.